files/image_config/procdockerstatsd/procdockerstatsd

```diff
@@ -37,7 +37,7 @@ def log_error(msg, also_print_to_console=False):
     syslog.closelog()
 
 # ========================== ProcessDocker class ==========================
-class ProcDockerStats:
+class ProcDockerStatsd:
```

Comment:
No need to append a 'd' to the class name.

Reply:
Should I rename only the file and folder name in that case? If not could you please explain what all needs to be renamed

Reply:
All of the other changes look good. You can remove the 'd' from this line and line 205.

Reply:
removed